package.json

```diff
@@ -83,6 +83,7 @@
     "dompurify": "^2.3.8",
     "emoji-picker-react": "2.1.1",
     "emoji-toolkit": "^6.6.0",
+    "emojione": "^4.5.0",
```

Comment:
can't we use `emoji-toolkit` for render? These are the newest versions for emoji-one I think

Refer: https://github.com/joypixels/emoji-toolkit/blob/master/examples/JAVASCRIPT.md